ios/Capacitor/Capacitor/CAPAssetHandler.swift

```diff
@@ -0,0 +1,68 @@
+import Foundation
+import MobileCoreServices
+
+class CAPAssetHandler: NSObject, WKURLSchemeHandler {
+    
+    func webView(_ webView: WKWebView, start urlSchemeTask: WKURLSchemeTask) {
+        var startPath = ""
+        let url = urlSchemeTask.request.url!
+        let stringToLoad = url.path
+        let scheme = url.scheme
+
+        if scheme == "capacitor" {
+            startPath = Bundle.main.path(forResource: "public", ofType: nil)!
+            if stringToLoad.isEmpty || url.pathExtension.isEmpty {
+                startPath.append("/index.html")
+            } else {
+                startPath.append(stringToLoad)
+            }
+        } else {
+            if !stringToLoad.isEmpty {
+                startPath = stringToLoad
+            }
+        }
+
+        let localUrl = URL.init(string: url.absoluteString)!
+        let fileUrl = URL.init(fileURLWithPath: startPath)
+
+        do {
+            var data = Data()
+            if !stringToLoad.contains("cordova.js") {
+                data = try Data(contentsOf: fileUrl)
+            }
+            let mimeType = mimeTypeForExtension(pathExtension: url.pathExtension)
+            let expectedContentLength = data.count
+            let headers =  [
+            "Content-Type": mimeType
+            ]
+            let urlResponse = URLResponse(url: localUrl, mimeType: mimeType, expectedContentLength: expectedContentLength, textEncodingName: nil)
+            let httpResponse = HTTPURLResponse(url: localUrl, statusCode: 200, httpVersion: nil, headerFields: headers)
+            if url.pathExtension == "mp4" {
```

Comment:
Just a note to check for other video types like .mov

Reply:
Done, added all the audio and video extensions supported on iOS and tested that they worked.